index.js

```diff
@@ -91,6 +91,21 @@ const SwipeListener = function (element, options) {
       y.push(touches[i].y);
     }
 
+    const xs = x[0], xe = x[x.length - 1], // Start and end x-coords
+    ys = y[0], ye = y[y.length - 1];  // Start and end y-coords
+    
+    if (touches.length > 1) {
+      const swipeReleaseEventData = {
+        detail: {
+          x: [xs, xe],
+          y: [ys, ye]
+        }
+      };
+      
```

Comment:
Could there be a better event name? Such as 'swipeend' and 'swipecomplete'?

Reply:
`swipe` signifies `swipeend` and `swipecomplete` grammatically.

Also added a `swipecancel` which is fired if the minimum travel distance is not met.